packages/account-v2/src/containers/POAForm/DocumentSubmission/DocumentSubmission.tsx

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+import { useBreakpoint } from '@deriv/quill-design';
+import { StandaloneArrowUpFromBracketBoldIcon } from '@deriv/quill-icons';
+import { Text } from '@deriv-com/ui/dist/components/Text';
+import FormDocumentUploadField from '../../../components/FormFields/FormDocumentUploadField';
+import CommonMistakesExamples from '../CommonMistakeExample/CommonMistakeExample';
+import { getExampleImagesConfig } from '../CommonMistakeExample/CommonMistakeExampleConfig';
+
+const listItems = [
+    'Utility bill: electricity, water, gas, or landline phone bill.',
+    'Financial, legal, or government document: recent bank statement, affidavit, or government-issued letter.',
+    'Home rental agreement: valid and current agreement.',
+];
+
+const DocumentSubmission: React.FC = () => {
+    const { isMobile } = useBreakpoint();
+
+    return (
+        <div className='flex flex-col items-start w-full gap-1200 sm:w-auto'>
+            <div className='flex h-1200 gap-400 self-stretch sm:self-auto justify-center items-center sm:gap-[11px]'>
```

Comment:
U can omit `sm` pseudo class as Tailwind is mobile first

Reply:
Replied above.